README.md

```diff
@@ -70,7 +70,8 @@ Example add-on configuration:
     "password": "!secret glances_influxdb_password",
     "database": "glances",
     "prefix": "localhost",
-    "interval": 60
+    "interval": 60,
+    "protocol": "http"
```

Comment:
Shouldn't we make this a simple `ssl` boolean to make it more consistent across the other add-ons?

Reply:
It was my original plan, however ran into jq errors with it being nested.  I'll take another look today.

Reply:
Not sure what challenge I was having before, but changed to boolean and made optional.